README.md

```diff
@@ -105,6 +105,15 @@ SIMPLE_SERVER_HOST=<HTTPS URL>
 SIMPLE_SERVER_HOST_PROTOCOL=https
 ```
 
+In the server repo, open `app/views/api/manifests/show.json.jbuilder`. Change:
+```
+json.endpoint "#{ENV["SIMPLE_SERVER_HOST_PROTOCOL"]}://#{ENV["SIMPLE_SERVER_HOST"]}/api/"
+```
+to:
+```
+json.endpoint "<HTTPS URL>/api/"
+```
+
```

Comment:
@kpethtel the steps above regarding `.env.development.local` is an alternative to this manifest change, and likely a cleaner solution. Had you done that when you tested? Should we add a line to the previous section that says "... and restart your server"?

Reply:
I had tried that. I think I had restarted my server after, but I'll try again.

Reply:
I re-tested this. I still couldn't get it to work with the existing instructions. I did restart the server. But using the server-side fix without changing the android configs did work for me.